src/main/webapp/js/fruzhin-lib.js

```diff
@@ -0,0 +1,66 @@
+import 'https://unpkg.com/@chainsafe/libp2p-yamux/dist/index.min.js';
+import 'https://unpkg.com/@chainsafe/libp2p-noise/dist/index.min.js';
+import 'https://unpkg.com/@libp2p/websockets/dist/index.min.js';
+import 'https://unpkg.com/@libp2p/kad-dht/dist/index.min.js';
+import 'https://unpkg.com/@libp2p/identify/dist/index.min.js';
+import 'https://unpkg.com/@libp2p/bootstrap/dist/index.min.js';
+import 'https://unpkg.com/@libp2p/ping/dist/index.min.js';
+import 'https://unpkg.com/@chainsafe/libp2p-gossipsub/dist/index.min.js';
+import 'https://unpkg.com/libp2p/dist/index.min.js';
+import 'https://unpkg.com/it-pipe/dist/index.min.js';
+import 'https://unpkg.com/it-pb-stream/dist/index.min.js';
+
+import * as Blake2b from './blake2b.js';
+import * as ED25519 from './ed25519.js';
+import * as HTTP from './http.js';
+import * as Fruzhin from './fruzhin.js'
+
+var startLibp2p = async (bootnodes) => {
+    console.log('Starting libp2p node');
+    console.log(bootnodes);
```

Comment:
These were removed [here](https://github.com/LimeChain/Fruzhin-Web/commit/9670bd3683e047f8aa1ffa85d3fa4788d7d3046b). I guess we don't need the anymore.

Reply:
Fixed after merging dev into this branch